easybuild/easyconfigs/p/Python/Python-2.7.10-GCC-4.9.3-2.25-bare.eb

```diff
@@ -0,0 +1,31 @@
+name = 'Python'
+version = '2.7.10'
+versionsuffix = '-bare'
+
+homepage = 'http://python.org/'
+description = "Python is a programming language that lets you work more quickly and integrate your systems more effectively."
+
+toolchain = {'name': 'GCC', 'version': '4.9.3-2.25'}
```

Comment:
we never settled that discussion, I think? this is more about keeping ncurses easyconfigs consistent